lib/TokenManager.ts

```diff
@@ -116,24 +119,9 @@ function get(storage, key) {
 }
 
 function getAsync(sdk, tokenMgmtRef, storage, key) {
```

Comment:
Is there a reason not to declare this function async?

Reply:
no real reason. it could be added, I don't think it would produce any side effect.

Reply:
I think it makes sense to declare `async` when `await` is included, but not for a function just return a promise.

Reply:
Why only then?  Async tells the IDE and someone skimming the code that the function returns a promise.  It's straight up information.

Reply:
By returning a promise, the function still can be called with await to resolve the value. But with `async` with the function, I'll expect the function to resolve a value instead of returning a promise. 
There is also some eslint to match async and await together. https://eslint.org/docs/rules/require-await#rule-details-260

Reply:
I don't follow.  That eslint rule just says "you may not have intended this", it doesn't actually explain that the practice is bad in any way (and like any eslint rule, likely an opposite rule exists).

async functions return a promise.  await converts a resolved value.  I can await promises, or I can use the promise directly, and I'll decide which based on what I'm doing.